src/js/utils/JobUtil.js

```diff
@@ -78,18 +78,27 @@ const JobUtil = {
       Object.assign(spec.run, { docker });
     }
 
+    // default values for id and policy
+    let scheduleId = "default";
+    let schedulePolicy = "ALLOW";
+    // preserve id and concurrencyPolicy
+    if (typeof spec.schedules != "undefined" && spec.schedules.length > 0) {
```

Comment:
@kensipe as discussed - let's adjust to `Array.isArray(spec.schedules)` instead of `typeof spec.schedules != "undefined"`.